bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
This is a debugging line so you can see the buglist query that is equivalent to the xmlrpc query. It's useful. I used it to verify my time-based filtering worked. So please just let this change be applied otherwise the next person will click the link in their console (assuming they have a console which makes URLs clickable, like mine) and get the WRONG query results.